main.yml

```
@@ -454,6 +458,8 @@ image-builder:
         matcher: isNotEmpty
   frontend:
     section: operations
+    module: image_builder#./RootApp
```

The `image-builder` app has dash in the name. So we have to slightly tweak the `module` defiitin
```suggestion
    module:
      appName: image-builder
      scope: imageBuilder
      module: ./RootApp
```

---

Right but I defined [the module](https://github.com/osbuild/image-builder-frontend/blob/main/config/prod.webpack.config.js) as `image_builder`, as a module name with a `-` wasn't supported. So it shuold be `image_builder` no?

---

Oh, so the `appName` should be `image-builder` and `scope` should be `image_builder`